.gitignore

```diff
@@ -5,6 +5,8 @@ install_manifest.txt
 bin/
 test/
 build/
+build-gcc/
```

Comment:
May I add these? It makes things easier when testing for each compiler.

Reply:
Not a fan of this.

Reply:
What can I do instead? Without this, I was having to do:

```
CC=gcc meson build
rm -rf build
...
CC=clang meson build
...
rm -rf build
```

Reply:
I'd prefer something like `/build-*` instead.